sql/bcnm_info.sql

```diff
@@ -45,262 +46,262 @@ CREATE TABLE `bcnm_info` (
 set autocommit=0;
-INSERT INTO `bcnm_info` VALUES (0,139,'rank_2_mission','Shitstain',1,18,1800,25,6,0,5);
-INSERT INTO `bcnm_info` VALUES (1,139,'tails_of_woe','Shitstain',1,18,1800,40,6,0,15);
-INSERT INTO `bcnm_info` VALUES (2,139,'dismemberment_brigade','Shitstain',1,18,1800,60,6,0,15);
-INSERT INTO `bcnm_info` VALUES (3,139,'the_secret_weapon','Shitstain',1,18,1800,99,6,0,5);
-INSERT INTO `bcnm_info` VALUES (4,139,'hostile_herbivores','Shitstain',1,18,1800,50,6,14,15);
-INSERT INTO `bcnm_info` VALUES (5,139,'shattering_stars','Shitstain',1,18,600,99,1,0,22);
-INSERT INTO `bcnm_info` VALUES (6,139,'shattering_stars','Shitstain',1,18,600,99,1,0,22);
-INSERT INTO `bcnm_info` VALUES (7,139,'shattering_stars','Shitstain',1,18,600,99,1,0,22);
-INSERT INTO `bcnm_info` VALUES (8,139,'carapace_combatants','Shitstain',1,18,900,30,3,0,15);
-INSERT INTO `bcnm_info` VALUES (9,139,'shooting_fish','Shitstain',1,18,900,20,3,0,15);
-INSERT INTO `bcnm_info` VALUES (10,139,'dropping_like_flies','Shitstain',1,18,1800,30,6,12,15);
-INSERT INTO `bcnm_info` VALUES (11,139,'horns_of_war','Shitstain',1,18,1800,0,18,11,15);
-INSERT INTO `bcnm_info` VALUES (12,139,'under_observation','Shitstain',1,18,900,40,3,16,15);
-INSERT INTO `bcnm_info` VALUES (13,139,'eye_of_the_tiger','Shitstain',1,18,900,50,3,0,15);
-INSERT INTO `bcnm_info` VALUES (14,139,'shots_in_the_dark','Shitstain',1,18,900,60,3,0,15);
-INSERT INTO `bcnm_info` VALUES (15,139,'double_dragonian','Shitstain',1,18,1800,0,6,15,15);
-INSERT INTO `bcnm_info` VALUES (16,139,'todays_horoscope','Shitstain',1,18,1800,0,6,0,15);
-INSERT INTO `bcnm_info` VALUES (17,139,'contaminated_colosseum','Shitstain',1,18,1800,0,6,17,15);
-INSERT INTO `bcnm_info` VALUES (18,139,'kindergarten_cap','Shitstain',1,18,900,0,3,0,15);
-INSERT INTO `bcnm_info` VALUES (19,139,'last_orc-shunned_hero','Shitstain',1,18,1800,0,6,0,15);
-INSERT INTO `bcnm_info` VALUES (20,139,'beyond_infinity','Shitstain',1,18,600,0,6,0,6);
-INSERT INTO `bcnm_info` VALUES (32,140,'save_the_children','Shitstain',1,18,600,0,6,0,5);
-INSERT INTO `bcnm_info` VALUES (33,140,'holy_crest','Shitstain',1,18,600,75,6,0,7);
-INSERT INTO `bcnm_info` VALUES (34,140,'wings_of_fury','Shitstain',1,18,900,20,3,34,15);
-INSERT INTO `bcnm_info` VALUES (35,140,'petrifying_pair','Shitstain',1,18,900,30,3,35,15);
-INSERT INTO `bcnm_info` VALUES (36,140,'toadal_recall','Shitstain',1,18,1800,30,6,36,15);
-INSERT INTO `bcnm_info` VALUES (37,140,'mirror_mirror','Shitstain',1,18,900,40,3,0,7);
-INSERT INTO `bcnm_info` VALUES (64,144,'rank_2_mission','Shitstain',1,18,1800,25,6,0,5);
-INSERT INTO `bcnm_info` VALUES (65,144,'worms_turn','Shitstain',1,18,1800,40,6,0,15);
-INSERT INTO `bcnm_info` VALUES (66,144,'grimshell_shocktroopers','Shitstain',1,18,1800,60,6,0,15);
-INSERT INTO `bcnm_info` VALUES (67,144,'on_my_way','Shitstain',1,18,1800,99,6,0,5);
-INSERT INTO `bcnm_info` VALUES (68,144,'thief_in_norg','Your mother',1,18,1800,99,6,0,5);
```

Comment:
wtf

Reply:
It was demolish. Except "your mother" I think was Autkast.

Reply:
Represent